gorm/order.go

```diff
@@ -85,8 +85,8 @@ func (o *Order) GetDrinkRow() *OrderRow {
 // GetAllOrderFromUser : return all Orders that Users have placed
 func GetAllOrderFromUser(id uint) (*[]Order, error) {
 	var orders []Order
-	err := DB.Preload("User").Preload("OrderRows").Preload("OrderRows.Product").
-		Where("user_id = ?", id).Find(&orders).Error
+	err := DB.Preload("User").Preload("OrderRows").Preload("OrderRows.Product").Preload("Destination").
+		Where("user_id = ? AND status_code >= ?", id, OrderStatusPlaced).Find(&orders).Error
```

Comment:
@dpham273 Notice in this place i changed the query (was `AND status_code != 1`). Because we want to select all orders which is *placed*. Orders can be of different state before *placed* and its status can be other than 1.